### PR TITLE
Javascript javascript2 week3

### PR DESCRIPTION
**Pull request by Impressiv3**

### CODE REVIEW

File: javascript/javascript2/week3/index.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+  coordinateListItemAccuracy.innerHTML = `The accuracy : ${accuracy}`;
+  coordinateList.appendChild(coordinateListItemAccuracy);
+
+  showGoogleMaps(currentLatitude, currentLongtitude);
+}
+
+/////////////////////////////////////////////////////////////////////////
+// 6. Optional Now show that location on a map using fx the Google maps api
+
+function showGoogleMaps(latitude, longitude) {
```
Comment (Lupescua) [inline javascript/javascript2/week3/index.js:110]: this part does not seem to work. 

I tried this solution  https://developers.google.com/maps/documentation/javascript/adding-a-google-map#maps_add_map-javascript

File: javascript/javascript2/week3/index.js [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+function checkForDoubleClick() {
+  console.log("double click!");
+}
+
+/////////////////////////////////////////////////////////////////////////
+// 9. Create a function called jokeCreator that has three parameters: shouldTellFunnyJoke - boolean,
+// logFunnyJoke - function and logBadJoke - function. If you set shouldTellFunnyJoke to true it should
+// call the logFunnyJoke function that should log out a funny joke. And vice versa.
+
+let shouldTellFunnyJoke = false;
```
Comment (Lupescua) [inline javascript/javascript2/week3/index.js:156]: :))) Good ones

**Review by Lupescua — COMMENTED** [review]
Awesome job bro!
Real clean code. 

I specially commend you on your use of comments. You're obviously real good and if you keep commenting your code like this, everyone will want to have you on their team! 

Let the job offers start coming your way!! :D

File: javascript/javascript2/week3/index.js (hunk 1 above)
Comment (Infamou5) [reply javascript/javascript2/week3/index.js:111]: https://maps.googleapis.com/maps/api/js/AuthenticationService.Authenticate?1shttp://127.0.0.1:5500/index.html&4sKEYHASBEENREMOVEDBYME&callback=_xdc_._6ntu5o&key=KEYHASBEENREMOVEDBYME&token=104540

==>> The reason why it doesn't want to work cause I used my own google API key when I was writing the code.
For obvious reasons I have not published my API key here on Github.  ==> KEYHASBEENREMOVEDBYME

File: javascript/javascript2/week3/index.js [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+});
+
+/////////////////////////////////////////////////////////////////////////
+// 4. Create two functions and assign them to two different variables.
+// One function logs out Earth, the other function logs out Saturn.
+// Now create a new third function that has one parameter: planetLogFunction.
+//  The only thing the third function should do is call the provided parameter function.
+// Try call the third function once with the Earth function and once with the Saturn function.
+
+let earthLogger = () => {
```
[the remark this answers is not in the document]
Comment (Infamou5) [reply javascript/javascript2/week3/index.js]: Changed / Fixed / Done !

File: javascript/javascript2/week3/index.js [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+/////////////////////////////////////////////////////////////////////////
+// 3. Create a button in html. When clicking this button,
+// use the function you created in the previous task to log out the text:
+// Called after 5 seconds 5 seconds after the button is clicked.
+
+const delayButton = document.getElementById("delay-button");
+const locationButton = document.getElementById("location-button");
+
+delayButton.addEventListener("click", () => {
+  delayedLogout("Called after 5 seconds", delayTime);
```
[the remark this answers is not in the document]
Comment (Infamou5) [reply javascript/javascript2/week3/index.js:42]: Changed / Fixed / Done !

**Comment by Infamou5** [comment]
All suggestions implemented.
Fixed all bugs. 
Ready to merge.

**Pull request merged**